lib/csv_importer/header.rb

```diff
@@ -9,6 +9,9 @@ class Header
 
     def columns
       column_names.map do |column_name|
+        # ensure column name escapes invisible characters
+        column_name = column_name.gsub(/\P{Print}|\p{Cf}/, '')
```

Comment:
Cool! It's the first time that I see `\P{Print}` and `\p{Cf}`. How does it differ from `.gsub(/[^[:print:]]/,'')`? Is the latter more generic or ruby-esque?

Reply:
```suggestion
        column_name = column_name.gsub(/[^[:print:]]/,'')
```